GrocyScanner.Service/Shared/Components/AddProductDialog.razor

```diff
@@ -29,27 +29,34 @@
                         Margin="Margin.Dense"
                         MinDate="DateTime.Today"
                         DateFormat="dd.MM.yyyy"/>
+                    @if (Product.Name == "Unknown")
```

Comment:
I think this is a bad way of checking if a product is unknown.
- There could be a product named exactly like this. I know, it's unlikely but not unreasonable to assume.
- If the "unknown" constant is changed in the code elsewhere, this would break. You can fix this by using a const. Then the compiler will throw if you attempt to set it and it will always be in sync with the other files.

Reply:
I've replaced it with a GUID const string. It's still hacky but I didn't want to add a nullable property or boolean to Product, that seemed worse. Let me know if this fix is acceptable